### DISCUSSION TITLE
WebRTC Sync stopped working

### DISCUSSION THREAD

**Discussion by sduddupudi** [lobehub/lobehub#2763] [General Question] [answered]
It's been a while since I started noticing an issue with WebRTC Sync. It shows a connection failed. It used to work. It's sad that I can't sync across my devices now.

**Comment by heltonteixeira** [comment]
Same here.

**Comment by arvinxx** [comment, marked as answer]
Check the document here: https://lobehub.com/docs/self-hosting/advanced/webrtc
